frontend/src/Components/Map/Inputs/MonthFilter.tsx

```diff
@@ -46,9 +49,12 @@ interface MonthFilterProps {
 const MonthFilter: React.FC<MonthFilterProps> = ({
   onEndChange,
   onStartChange,
+  minDate,
+  maxDate,
 }) => {
-  const [startDate, setStartDate] = useState(new Date('2022/06/08'));
-  const [endDate, setEndDate] = useState(new Date());
+  const [startDate, setStartDate] = useState<Date>(new Date('2022/06/08'));
+  const [endDate, setEndDate] = useState<Date>(new Date('2022/09/08'));
```

Comment:
Can't get this to to show minDate and maxDate, because it takes a while for it get the data and figure stuff out -> ends up leaving the buttons tiny. Not sure how to fix